docs/release_notes/0.42.0.md

```diff
@@ -0,0 +1,15 @@
+# Release 0.42.0
+
+## Features
+Add default security group rules to custom SGs (#3027)
+
+## Improvements
+Disallow setting internal fields (#3450)
+
+## Bug Fixes
+Force addons during cluster creation (#3438)
+Fix: nil pointer deref when vpc is empty (#3447)
```

Comment:
This bug didn't make it to any release because it was introduced by #3027. It may be confusing to users if we include it. We should still give attribution to the contributor (@liranp) though.

Reply:
ah yes good point. i will remove it and leave his name in